nixarr/default.nix

```diff
@@ -253,6 +254,33 @@ in {
         example = [46382 38473];
       };
     };
+
+    monitoring = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        example = true;
+        description = ''
+          Whether to enable Prometheus monitoring for nixarr services.
+          This will configure various exporters for services that support them.
+        '';
+      };
+
+      exporters = mkOption {
+        type = types.attrsOf types.bool;
```

Comment:
I would prefer for this to be more explicit, i.e. each supported service has it's own option (also so it's integrated into the option-docs).

Reply:
Sure, makes sense.